cli/medperf/commands/mlcube/mlcube.py

```diff
@@ -28,6 +29,27 @@ def list(
     )
 
 
+@app.command("create")
+@clean_except
+def create(
+    template: str = typer.Argument(
+        None,
+        help=f"MLCube template name. Available templates: [{' | '.join(config.templates.keys())}]",
```

Comment:
this should be a required argument, right?

Reply:
You're right! I'll make the required change